### DISCUSSION TITLE
3.3.x

### DISCUSSION THREAD

**Discussion by nicolas377** [GeoFS-Extensions/autoland#112] [Updates]
This discussion is a open forum for discussing the 3.3.x series of updates.

## Current status
As of 11-2-2021, we're just about ready to push to production. We have one issue that's holding us from pushing to production. @Guy-Adler is working on it locally. For now, we're planning future updates.

## 3.3.0
- Changes:
    - [x] **Add support for multiple key press keybinds** (new script: Keyboard Mapping)
    - [x] **Move the entire codebase into one monorepo** #81 
    - [x] *Switch script optimization to webpack* #120
    - [x] *Style type names to UpperCamelCase* #122 
    - [x] <ins>Scripts not loading</ins> #118
    - [x] <ins>FMC uses data from previous routes when fields are empty</ins> #87
    - [x] <ins>Spoiler arming keybind requires SHIFT, ALT or CTRL</ins> #127 
    - [x] <ins>Keybinds not stored in localStorage correctly</ins> #128
    - [x] <ins>Can't remap keydown/keyup keybinds</ins> #129

## 3.3.1
- Changes:
    - [x] Fix fmc routes #136

**Comment by nicolas377** [comment]
#120 might be something we want to add here, since it would increment the script's minor versions. I'm thinking we add that as 3.3.1 and bump the others up. Opinions?

**Reply by nicolas377** [reply]
I'm going to go ahead and add this as 3.3.1 and bump #36 to 3.3.2, and #64 to 3.3.3.

**Comment by nicolas377** [comment]
I think that all the changes we want to add should be in patch 0 updates (e.g. 3.3.**0**). For us, that would mean bumping #36 to 3.3.0, and I'm thinking that #64 should be 3.4.0. Patches should only be for bugs discovered after the release of the patch 0 update.

@Guy-Adler what do you think of this?

**Reply by Guy-Adler** [reply]
Sounds good

**Reply by nicolas377** [reply]
Alright! I'll go ahead and get that set up.
